src/main/java/org/example/coursework2/repository/impl/JavaQuestionRepository.java

```diff
@@ -0,0 +1,52 @@
+package org.example.coursework2.repository.impl;
+
+
+import org.example.coursework2.entity.Question;
+import org.example.coursework2.exception.NotUniqueQuestionException;
+import org.example.coursework2.repository.QuestionRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Repository
+public class JavaQuestionRepository implements QuestionRepository {
+    private final List<Question> questions = new ArrayList<>();
+
+    @Override
+    public Question aaddQuestion(Question question) {
+        return null;
+    }
+
+    @Override
+    public Question removeQuestion(Question question) {
+        return null;
+    }
```

Comment:
Нужно добавить реализацию:
```suggestion
    @Override
    public Question aaddQuestion(Question question) {
        return add(question);
    }

    @Override
    public Question removeQuestion(Question question) {
        return remove(question);
    }
```